src/main/java/us/kilroyrobotics/Constants.java

```diff
@@ -135,4 +135,15 @@ public static final class CoralMechanismConstants {
         /** Speed of the motor while the robot is in motion to keep the coral piece in place */
         public static final double kWheelSpeedHolding = 0.15;
     }
+
+    public static final class AlgaeConstants {
+
+        /** Algae motor IDs */
+        public static final int kAlgaeMotorLeaderId = 45;
+
+        public static final int kAlgaeMotorFollowerId = 46;
+
+        public static final double kAlgaeSpeedIntaking = 0.75;
```

Comment:
We should switch to using velocity control at some point

Reply:
Maybe, but for now it's just fast in and fast out which is working.

Reply:
I know. But i want us to learn how to use velocity control